src/common/Guardian.sol

```diff
@@ -115,6 +117,45 @@ contract Guardian is IGuardian {
         multiAdapter.disputeRecovery(centrifugeId, adapter, hash);
     }
 
+    /// @inheritdoc IGuardian
+    function wireAdapters(uint16 centrifugeId, IAdapter[] calldata adapters) external onlySafe {
+        multiAdapter.file("adapters", centrifugeId, adapters);
```

Comment:
On second thought, would similarly check here that `multiAdapter.adapters(centrifugeId).length == 0`